lnbits/extensions/discordbot/output.file

```diff
@@ -0,0 +1,15 @@
+from fastapi import Request
+from fastapi.params import Depends
+from starlette.responses import HTMLResponse
+
+from lnbits.core.models import User
+from lnbits.decorators import check_user_exists
+
+from . import discordbot_ext, discordbot_renderer
+
+
+@discordbot_ext.get("/", response_class=HTMLResponse)
+async def index(request: Request, user: User = Depends(check_user_exists)):
+    return discordbot_renderer().TemplateResponse(
+        "discordbot/index.html", {"request": request, "user": user.dict()}
+    )
```

Comment:
What is this file doing?

Reply:
Hello, thanks for checking.

I have just copied usermanager nearly entirely and edited so it works with discord ids. It's a copy from https://github.com/lnbits/lnbits-legend/pull/258. I can't tell you what the code does in detail, but it works again..

Reply:
this didn't exist in the earlier code and I have no idea what it is 😅 Doesn't look functional/used

Reply:
Oh well i checked it again, guess it was something which was created by testing some renaming with sed. Removed that file. Sorry for that coding mess ...

Reply:
All good :)